tests/scripts/cluster-scale-down-test.sh

```diff
@@ -97,12 +96,19 @@ if [ $clientexit -ne 0 ]; then
     exit $clientexit
 fi
 
-# Check the output from clusterclient
-expected="bar1
+# Check the output from clusterclient, which depends on timing.
+
+# Client sends the command 'GET {foo}2' just after node2 closes its socket.
```

Comment:
The node is called `nodeid2` in cluster slots. In other comments it is called `node #2`. Not that it matters, but it's nicer to be consistent.

Reply:
There is a lot of comments regarding nodeidX in this test so will go with that here.